spec/models/ems_refresh/parsers/kubernetes_spec.rb

```diff
@@ -49,9 +49,25 @@
                        :image      => {:name => "subname/example", :tag => nil, :image_ref => example_ref},
                        :registry   => {:name => "host.com", :host => "host.com", :port => nil}},
 
+                      {:image_name => "host.com/example",
+                       :image      => {:name => "example", :tag => nil, :image_ref => example_ref},
+                       :registry   => {:name => "host.com", :host => "host.com", :port => nil}},
+
                       {:image_name => "host.com:1234/subname/more/names/example:tag",
                        :image      => {:name => "subname/more/names/example", :tag => "tag", :image_ref => example_ref},
-                       :registry   => {:name => "host.com", :host => "host.com", :port => "1234"}}]
+                       :registry   => {:name => "host.com", :host => "host.com", :port => "1234"}},
+
+                      {:image_name => "localhost:1234/name",
+                       :image      => {:name => "name", :tag => nil, :image_ref => example_ref},
+                       :registry   => {:name => "localhost", :host => "localhost", :port => "1234"}},
+
+                      {:image_name => "localhost:1234/name@sha256:1234567abcdefg",
+                       :image      => {:name => "name", :tag => "sha256:1234567abcdefg", :image_ref => example_ref},
```

Comment:
> In docker nomenclature that's called digest, it should probably be another column. I believe that could be a db migration in the future (when we'll want to be precise about it
> Ah, I assumed the digest and the image_ref would be the same.

@zeari they are but there's a slightly different connotation, one (digest) is what the user requested and the image_ref is what is actually running. Except for mistakes of the infrastructure they are the same. Nonetheless it's interesting to know that the user specifically requested a certain digest.

Also think of how you're re-building the `full_name` now... is the digest handled? There's probably an issue there (just because you're not differentiating between tags and digests)